prime-router/metadata/hl7_mapping/ORU_R01/base/specimen.yml

```diff
@@ -22,20 +22,40 @@ elements:
     schema: datatype/ei-entity-identifier
     resourceIndex: entityIdIndex
 
-  - name: type
```

Comment:
In PR 6881 you are removing specificity of identifiers - IE this would just be 'type'. Why add it here? (same question for rest of file)

Reply:
I don't follow.  Will ask you more info about this.

Reply:
After the discussion, this is fine.